### DISCUSSION TITLE
Do I need to gitignore certificate files or any files in the ios/builds folder?

### DISCUSSION THREAD

**Discussion by julianfssen** [fastlane/fastlane#19121] [Q&A] [answered]
Hi! Thanks for the awesome tool :)

I deployed a React Native app on iOS via Fastlane today without any issues. However, I noticed that the process generated a `ios/builds` folder with a few certificate files ending in `.cer`, `.p12`, and `.certSigningRequest`. I also noticed a huge number of asset files being included as well like these:

![image](https://user-images.githubusercontent.com/25947043/126490005-32129aa1-2cda-47f2-9457-947c33686dda.png)

May I know if I need to commit these files or is it alright to gitignore the `ios/builds` folder entirely? Thanks again!

**Comment by crazymanish** [comment, marked as answer]
Hi @julianfssen 

You are looking inside `xxxxxx.archive` which contains everything and is normal. If you will try to do Archive using Xcode, and try to see inside xxxxx.archive, you should see the same.


Regarding the `.gitignore`, I will highly recommend ignoring the entire `ios/builds` folder entirely otherwise your git-repo size will increase with crazy speed 😉 

I hope this will help! 
Please let me know in case of any issue!

**Reply by julianfssen** [reply]
Hi @crazymanish ! Thanks for explaining that and yes it clears my questions, appreciate it! :)
